app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/wallet/WalletInstalledScreen.kt

```diff
@@ -0,0 +1,194 @@
+  val launcher = rememberLauncherForActivityResult(
+    contract = StartActivityForResult()
+  ) {
+    onFinish(it.resultCode == Activity.RESULT_OK)
+  }
+
+  val onRedirect: () -> Unit = {
+    uri?.let {
+      val intent = Intent(Intent.ACTION_VIEW).setPackage("com.appcoins.wallet")
+      intent.data = uri
+
+      try {
+        launcher.launch(intent)
+      } catch (e: Exception) {
+        onFinish(false) // TODO: Show error view with close button
+      }
+    }
+  }
+
+  var launched by remember { mutableStateOf(false) }
+
+  LaunchedEffect(Unit) {
+    delay(3000)
+    if (!launched) onRedirect()
+    launched = true
+  }
+
+  AppGamesPaymentBottomSheet(
+    onClick = {
+      launched = true
+      onRedirect()
+    },
+    onOutsideClick = {
+      onFinish(false)
+    }
+  ) {
+    val configuration = LocalConfiguration.current
+    when (configuration.orientation) {
+      Configuration.ORIENTATION_LANDSCAPE -> WalletInstalledLandscapeView()
+      else -> WalletInstalledPortraitView()
+    }
+  }
+}
+
+@Composable
+fun WalletInstalledPortraitView() {
+  Column(
+    modifier = Modifier
+      .fillMaxWidth()
+      .wrapContentHeight(),
+    horizontalAlignment = Alignment.CenterHorizontally,
+  ) {
+    Image(
+      modifier = Modifier.padding(top = 40.dp),
+      imageVector = getWalletInstalled(
+        Palette.AppCoinsPink,
+        Palette.White,
+        Palette.Primary,
+        Palette.Black
+      ),
+      contentScale = ContentScale.FillBounds,
+      contentDescription = null
+    )
+    Text(
+      modifier = Modifier.padding(top = 54.dp),
+      text = stringResource(R.string.iap_appc_wallet_install_success_title),
+      style = AGTypography.Title,
+      textAlign = TextAlign.Center,
+      color = Palette.Black
+    )
+    Text(
+      modifier = Modifier.padding(top = 12.dp, bottom = 88.dp),
+      text = stringResource(R.string.iap_appc_wallet_install_success_body),
+      style = AGTypography.DescriptionGames,
+      textAlign = TextAlign.Center,
+      color = Palette.Black
+    )
+  }
+}
+
+@Composable
+fun WalletInstalledLandscapeView() {
+  Column(
+    modifier = Modifier
+      .fillMaxWidth()
+      .wrapContentHeight()
+      .verticalScroll(rememberScrollState()),
+    horizontalAlignment = Alignment.CenterHorizontally,
+  ) {
+    Image(
+      modifier = Modifier.padding(top = 40.dp),
+      imageVector = getWalletInstalled(
+        Palette.AppCoinsPink,
+        Palette.White,
+        Palette.Primary,
+        Palette.Black
+      ),
+      contentScale = ContentScale.FillBounds,
+      contentDescription = null
+    )
+    Text(
+      modifier = Modifier.padding(top = 16.dp, start = 10.dp, end = 10.dp),
+      text = stringResource(R.string.iap_appc_wallet_install_success_title),
+      style = AGTypography.Title,
+      textAlign = TextAlign.Center,
+      color = Palette.Black
+    )
+    Text(
+      modifier = Modifier.padding(top = 6.dp, start = 10.dp, end = 10.dp, bottom = 10.dp),
```

Comment:
The bottom padding should be a bit larger because it has issues where the text is cut on the smaller device previews, even when the view is scrollable